source/connect/favorite-connections/import-export.txt

```diff
@@ -20,15 +20,20 @@ use them yourself in your other workspaces.
 
    By default, when you export favorite connections, database usernames
```

Comment:
I can't comment directly on the intro, but consider adding a brief sentence that mentions the two methods you can use to do the export / import. Something like:

"You can manage favorite connections in the Compass UI or with the Compass command-line interface (CLI)."

Reply:
Updated.